### PR TITLE
chore: migrate to new prow reviewing

### PR DESCRIPTION
**Pull request by wuhuizuo**
- Backgroup: https://internals.tidb.io/t/topic/854
- Ref: https://github.com/ti-community-infra/configs/issues/824

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [x] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [x] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```
Comment (csuzhangxc) [inline OWNERS:1]: need to be updated to match the newest content in https://github.com/pingcap/community/blob/master/teams/kubernetes/membership.json, right?

Comment (wuhuizuo) [reply OWNERS:1]: yes, you are right, the PR is outdated, I will update it.

Comment (wuhuizuo) [reply OWNERS:1]: updated

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**